dll/win32/netid/lang/ro-RO.rc

```diff
@@ -2,93 +2,93 @@
-    10 "Noul nume de calculator ""%1"" este prea lung. Numele nu poate fi mai lung de 63 de caractere."
-    11 "Noul nume de calculator introdus nu e formatat corect. Numele standard pot conține litere (a-z, A-Z), numere (0-9), și cratime (-), dar nu spații sau puncte (.). Numele nu poate fi compus numai din cifre."
-    22 "Bun venit în grupul de lucru „%1”."
-    23 "Bun venit în domeniul „%1”."
-    24 "Repornirea calculatorului este necesară pentru a aplica schimbările aduse."
-    25 "Puteți schimba numele și apartenența acestui calculator la un domeniu sau grup de lucru. Schimbările pot afecta accesul la resursele rețelei."
-    1021 "Notă: Doar administratorii pot schimba identitatea calculatorului."
-    1022 "Notă: Identitatea calculatorului nu poate fi schimbată deoarece:"
-    1029 "Noul nume de calculator ""%1"" e un număr. Numele nu poate fi un număr."
-    1030 "Noul nume „%1” conține caractere nepermise. Caracterele nepermise sunt ` ~ ! @ # $ %% ^ & * ( ) = + [ ] { } \\ | ; : ' "" , < > / și ?"
-    1031 "Primul domeniu al numelui de domeniu ""%1"" este un număr. Primul domeniu nu poate fi un număr."
-    1032 "Numele de domeniu ""%1"" conține caractere ce nu sunt permise. Numele standard de DNS pot conține litere (a-z, A-Z), numere (0-9), și cratime, dar nu spații. Punctele (.) sunt folosite la domeniile separate. De exemplu: domeniu-1.reactos.org."
+    7 "Numele domeniului ""%1"" nu este conform specificaţiilor serviciului Internet Domain Name, deşi este un nume ReactOS legal. Pentru nume nestandard utilizaţi serverul DNS ReactOS."
+    8 "Numele de domeniu ""%1"" nu este formatat corespunzător.  Punctele (.) sunt utilizate pentru a separa domeniile.  Fiecare domeniu este limitat la %2!d! caractere. Exemplu: domeniul-1.reactos.org."
+    10 "Noul nume al computerului ""%1"" este prea lung. Numele nu poate fi mai lung de 63 de caractere."
+    11 "Numele nou introdus pentru computer nu este formatat corespunzător. Numele standard pot conţine litere (a-z, A-Z), cifre (0-9) şi cratime (-), nu şi spaţii sau puncte (.). Numele nu poate fi compus în întregime numai din cifre."
+    22 "Bun venit la grupul de lucru ""%1""."
+    23 "Bun venit la domeniul ""%1""."
+    24 "Pentru ca modificările să aibă efect reporniţi computerul."
+    25 "Aveţi posibilitatea să modificaţi numele şi calitatea de membru al acestui computer. Modificările pot afecta accesul la resursele reţelei."
+    1021 "Notă: Numai administratorii pot modifica elementele de identificare pentru acest computer."
+    1022 "Notă: Imposibil de modificat elementele de identificare pentru acest computer, deoarece:"
+    1029 "Numele nou ""%1"" al computerului este un număr.  Numele nu poate fi un număr."
+    1030 "Numele nou ""%1"" al computerului conţine caractere nepermise. Caracterele care nu sunt permise includ ` ~ ! @ # $ % ^ & * ( ) = + [ ] { } \\ | ; : ' \" , < > / şi ?"
```

Comment:
The MSVC compiler does not compile due to
`D:\a\reactos\reactos\src\dll\win32\netid\lang/ro-RO.rc(87) : error RC2148 : expected numeric constant in string table`
I do hope that my suggestion helps (but have not tested it):
```suggestion
    1030 "Numele nou ""%1"" al computerului conţine caractere nepermise. Caracterele care nu sunt permise includ ` ~ ! @ # $ % ^ & * ( ) = + [ ] { } \\ | ; : ' "" , < > / şi ?"
```